### PR TITLE
AHHHH IDK

### PR DESCRIPTION
**Pull request by SelFerns**
Some merge conflict was resolved (axios), and idk , its working and up to date with your previous changes, no html, css, js, were harmed. Thank you. AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH.

### CODE REVIEW

File: xconnect_frontend/xconnect/public/index.html
Comment (SelFerns) [inline xconnect_frontend/xconnect/public/index.html:1]: Cool New Fonts (Fonts and Fonting fr fr)

File: xconnect_frontend/xconnect/src/modules/calendar/Calendar.js
Comment (SelFerns) [inline xconnect_frontend/xconnect/src/modules/calendar/Calendar.js:1]: Main thing... Have to integrate this with backend (Task fr fr)

File: xconnect_frontend/xconnect/src/modules/calendar/calendar.js
Comment (SelFerns) [inline xconnect_frontend/xconnect/src/modules/calendar/calendar.js:1]: THERE WAS SO MUCH WRONG HERE

File: xconnect_frontend/xconnect/src/modules/helpLine/helpline.js
Comment (SelFerns) [inline xconnect_frontend/xconnect/src/modules/helpLine/helpline.js:1]: I don't think there are any major changes here

File: xconnect_frontend/xconnect/src/modules/timetable/TimeTable.css
Comment (SelFerns) [inline xconnect_frontend/xconnect/src/modules/timetable/TimeTable.css:1]: AH-HAHA good css fr fr (acc to me atleast)

File: xconnect_frontend/xconnect/src/modules/timetable/timetable.js
Comment (SelFerns) [inline xconnect_frontend/xconnect/src/modules/timetable/timetable.js:1]: Tell Daniel to integrate this thing. Thank you.

File: xconnect_frontend/xconnect/src/pages/Home/components/HeroSection.js
Comment (SelFerns) [inline xconnect_frontend/xconnect/src/pages/Home/components/HeroSection.js:1]: I don't remember what i did here but something wasn't fine

File: xconnect_frontend/xconnect/src/modules/canteen/CDashboard/shared/Sidebar.js
[the remark this answers is not in the document]
Comment (sea-rod) [reply xconnect_frontend/xconnect/src/modules/canteen/CDashboard/shared/Sidebar.js:1]: manoj

**Review by sea-rod — APPROVED** [review]

**Pull request merged**